src/app/Scenes/InfiniteDiscovery/InfiniteDiscovery.tsx

```diff
@@ -221,6 +165,71 @@ const InfiniteDiscoverySpinner: React.FC = () => (
 
 export const InfiniteDiscoveryQueryRenderer = () => <InfiniteDiscovery />
 
+type InfiniteDiscoveryArtwork = NonNullable<
+  NonNullable<
+    NonNullable<NonNullable<InfiniteDiscoveryQuery$data["discoverArtworks"]>["edges"]>[0]
+  >["node"]
+>
+interface InfiniteDiscoveryArtworkCardProps {
+  artwork: InfiniteDiscoveryArtwork
+}
+
+const InfiniteDiscoveryArtworkCard: React.FC<InfiniteDiscoveryArtworkCardProps> = ({ artwork }) => {
+  const { color } = useTheme()
+  const { width: screenWidth } = useScreenDimensions()
+
+  const src = !!artwork?.images?.[0]?.url ? artwork.images[0].url : undefined
+  const width = !!artwork?.images?.[0]?.width ? artwork.images[0].width : 0
+  const height = !!artwork?.images?.[0]?.height ? artwork.images[0].height : 0
+
+  const size = sizeToFit({ width: width, height: height }, { width: screenWidth, height: 500 })
+
+  return (
+    <Flex backgroundColor={color("white100")} width="100%" height={800}>
+      <EntityHeader
+        name={artwork?.artistNames ?? ""}
+        meta={artwork?.artists?.[0]?.formattedNationalityAndBirthday ?? undefined}
+        imageUrl={artwork?.artists?.[0]?.coverArtwork?.images?.[0]?.url ?? undefined}
+        initials={artwork?.artists?.[0]?.initials ?? undefined}
+        RightButton={
+          <Button variant="outlineGray" size="small">
+            Follow
+          </Button>
+        }
+        p={1}
+      />
+      <Spacer y={2} />
+
+      <Flex alignItems="center" backgroundColor={color("purple60")}>
```

Comment:
no need for this here
```suggestion
      <Flex alignItems="center" backgroundColor={"purple60"}>
```